apps/sv/frontend/src/__tests__/governance/forms/set-dso-rules-form.test.tsx

```diff
@@ -309,11 +309,79 @@ describe('Set DSO Config Rules Form', () => {
     await user.click(submitButton); // review proposal
     await user.click(submitButton); // submit proposal
 
-    waitFor(() => {
-      expect(screen.getByText('Action Required')).toBeDefined();
-      expect(screen.getByText('Inflight Votes')).toBeDefined();
-      expect(screen.getByText('Vote History')).toBeDefined();
-      expect(screen.getByText('Successfully submitted the proposal')).toBeDefined();
+    await waitFor(() => {
+      expect(screen.queryByText('Action Required')).toBeDefined();
+      expect(screen.queryByText('Inflight Votes')).toBeDefined();
+      expect(screen.queryByText('Vote History')).toBeDefined();
+      expect(screen.queryByText('Successfully submitted the proposal')).toBeDefined();
```

Comment:
Not related to this feature but needed to be fixed.

`getByText` throws an exception which causes this test to flake sometimes. We don't want unhandled exceptions while waiting for the condition to be true.